### ISSUE TITLE
Conflict Between `Alt + \` Shortcut and Backtick Character Input on Italian Keyboard

### ISSUE THREAD

**Issue by Gioxxy** [zed-industries/zed#17162]
### Check for existing issues

- [X] Completed

### Issue Description

When using an Italian keyboard, the character `` ` `` (backtick) is typically inserted by pressing `Alt + \`. However, in the current configuration, this key combination is bound to the following command:

```json
{
  "context": "Editor && !inline_completion",
  "bindings": {
    "alt-\\": "editor::ShowInlineCompletion"
  }
}
```

As a result, pressing `Alt + \` does not insert the backtick character as expected, but instead triggers the "Show Inline Completion" command. Attempting to reassign this shortcut to `Ctrl + \` in the key bindings override does not work.

### Expected Behavior

Pressing `Alt + \` should insert the backtick character `` ` ``, or an alternative solution should be provided to allow the user to easily input this character on an Italian keyboard without interfering with other key bindings.

### Actual Behavior

`Alt + \` is currently mapped to the "Show Inline Completion" command, preventing the insertion of the backtick character. Reassigning this command to another key combination (`Ctrl + \`) in the key bindings override does not work.

### Environment

Zed: v0.150.4 (Zed)
OS: macOS 14.6.1
Memory: 16 GiB
Architecture: aarch64

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by JunkuiZhang** [comment]
Could you please let me know your keyboard layout?
- Is it the Italian 104 layout or the standard Italian layout?
- Are you using an ISO or ANSI layout?
- What's the input method?

**Comment by Gioxxy** [comment]
**Layout Type:** Standard Italian Layout
**Keyboard Layout:** ISO
**Input Method:** Italian

**Comment by AurevoirXavier** [comment]
What I propose is that make Zed able to disable some keyboard shortcut like VSCode.

```
 {
     "command": "-workbench.action.togglePanel",
     "key": "..",
     "when": ".."
 }
```

In Zed, in could be something like:

```
{
    "context": "..",
    "free/loose": {
        "alt-\\": "editor::ShowInlineCompletion"
    }
},

**Comment by moubctez** [comment]
Same with Alt+Z on Polish keyboard. As a work-around, I had to add:

```
{
    "context": "Editor",
    "bindings": {
      "alt-z": ["workspace::SendKeystrokes", "ż"]
    }
}
```

+1 for an option to disable certain key bindings.

**Comment by SomeoneToIgnore** [comment]
There's a way to disable keybindings with
```
 "bindings": {
    "alt-\\": null
  }
```

and in fact, seems that we have a duplicate issue for this closed: https://github.com/zed-industries/zed/issues/12726

**Comment by Gioxxy** [comment]
> There's a way to disable keybindings with
> 
> ```
>  "bindings": {
>     "alt-\\": null
>   }
> ```
> 
> and in fact, seems that we have a duplicate issue for this closed: #12726

```
  {
    "context": "Editor && !inline_completion",
    "bindings": {
      "ctrl-\\": "editor::ShowInlineCompletion",
      "alt-\\": null
    }
  }
```
Yes, it works. Thanks!

**Comment by moubctez** [comment]
I couldn't find the `null` in the documentation https://zed.dev/docs/key-bindings
